vernac/comInductive.ml

```diff
@@ -283,13 +282,15 @@ let inductive_levels env evd arities inds =
         in
         let minlev =
           (* Indices contribute. *)
-          if indices_matter env && List.length ctx > 0 then (
```

Comment:
Why this seemingly unrelated change?

Reply:
Drive by cleanup while working to understand what's going on

Reply:
It's a bit hackish to rely on SProp being lower than anybody when there is no cumulativity relation between it and anybody else, though?

Reply:
Where do we rely on that?

Reply:
In ComInductive.sign_level.

Reply:
If you mean Universe.sup imo that's well defined on sprop (at the cost of losing `u <= max(u,x)` when x = sprop)

Reply:
Depends what you call well-defined. Since SProp is not in cumulativity relation with any other universe, `max(sprop, u)` shouldn't even be defined for any `u != SProp`.